ui-participant/src/hub/kit/KitTask.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react'
+import { KitRequest } from '../../api/api'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faTruckFast } from '@fortawesome/free-solid-svg-icons'
+
+/** Renders kit tasks for the hub page */
+export default function KitTask({ kitRequests }: {kitRequests: KitRequest[]}) {
+  const kitEventProps = {
+    padding: '1em 0em',
+    borderBottom: '1px solid #e4e4e4',
+    width: '100%',
+    color: '#595959'
+  }
+
+  const kitHeaderProps = {
+    borderBottom: '1px solid #e4e4e4',
+    width: '100%',
+    color: '#595959'
+  }
+
+  const iconProps = {
+    color: 'var(--brand-color)',
+    fontSize: '30px'
+  }
+
+  const iconClass = 'col-1'
+  const eventTextClass = 'col-8'
+  const eventDateClass = 'col-3 text-end'
+  const kitEventClass = 'row mb-3 pt-3'
+
+  const hasKitRequests = kitRequests.length > 0
+  if (!hasKitRequests) {
+    return null
+  }
+
+  const unreturnedKit = (kitRequest: KitRequest) => {
```

Comment:
I'd pick a name that more clearly indicates this is a function.  'isUnreturnedKit' or 'unreturnedKitFilter'